model/database/common.yaml

```diff
@@ -21,3 +21,26 @@ groups:
           The `error.type` SHOULD match the error code returned by the database or the client library,
           the canonical name of exception that occurred, or another low-cardinality error identifier.
           Instrumentations SHOULD document the list of errors they report.
+
+  - id: attributes.db.cosmosdb.minimal
+    type: attribute_group
+    brief: 'Azure Cosmos DB attributes'
+    attributes:
+      - ref: db.system
+        requirement_level: required
+      - ref: db.collection.name
+        requirement_level: required
+      - ref: db.namespace
+        requirement_level: required
+      - ref: db.operation.name
+        requirement_level: required
+      - ref: server.address
+        requirement_level: required
+      - ref: db.cosmosdb.status_code
+        requirement_level: required
+      - ref: db.cosmosdb.sub_status_code
+        requirement_level: required
+      - ref: db.cosmosdb.client_id
```

Comment:
this attribute potentially has a high cardinality. Let's say I create a client every few hours (my service restarts a lot) and I have ~100 instances of the service. I'm going to have ~1000 unique values of client-id per day.

If I do some weird things like creating a client per operation, it's going to definitely blow up my metrics - they will reach cardinality limit and will become useless.

Consider making it opt-in or removing from metrics.

Reply:
Ideally, customer create long live cosmos db client instances but it is a valid scenario where customer didn't follow the bad practice and decide to create instances again and again or service restarts a lot. Then these metrics would become useless but along with that, it will also show that something is wrong with the current application design.

@FabianMeiswinkel  @jcocchi @kirankumarkolli

Reply:
I'm not sure what's the goal of having this attribute on metrics. 

Let's look into possible scenarios:

1.  I'm a user who follows best practices, I create 1 client per db+container for the service instance lifetime. Then I'll have `service.instance.id` captured for my by otel as a resource attribute and will be able to distinguish clients by the service instance id.
2. I create several clients per db+container for the service lifetime then this could should me if one of the clients experience more issues than others.
3. I create tons of clients per service instance lifetime, my metrics are not usable and memory overhead of aggregating them in the process is huge. If cosmos SDK reported number of created connection and number of active connections, I could have looked at it and figured out that I'm creating too many of them and do something wrong.

It seems this attribute only helps with case 2 - is it worth it? What are the scenarios when you'd recommend users to break metrics down by the client id specifically?

Reply:
or actually, we can have separate metrics for client instance count, something like `db.cosmosdb.instance.count` which will show if client is creating multiple instances.
@lmolkova @jcocchi  @kirankumarkolli

Reply:
As you suggested removed, `client_id` as dimension and added `machine_id` as a dimension